### ISSUE TITLE
Drafts of department pages on Boston.gov timing out

### ISSUE THREAD

**Issue by duffy-james** [CityOfBoston/boston.gov-d7#1152]
#### Expected Behavior/Acceptance Criteria 
<br>
A user should be able to preview new drafts and drafted changes of a department profile page when making edits in Drupal for Boston.gov.

#### Actual Behavior
<br>
New drafts of department pages are timing out when you attempt to save changes. A draft ends up getting created, but you can't view the draft without timing out. A user is left to just publish pages in order to see changes.
<br><br>
One note: in testing, I've found that these content types are all working properly:

- program/initiative profiles
- guides
- landing pages
- articles
- person profiles
- events, and
- posts.

<br>
So, the issue appears limited to department profile content types.

#### Browser
<br> 
All browsers

#### Device
<br> 
All devices

#### Example URL
<br>
You can see the error when you attempt to save changes for the [Digital Team page](https://www.boston.gov/departments/digital-team).

#### Relevant Screenshot(s) or Logs
<br> 
![tech-difficulties](https://user-images.githubusercontent.com/7929556/43144808-04d61e34-8f2c-11e8-9845-5016731363df.PNG)
That's the error that appears are attempting to save.

#### Steps to Reproduce the Behavior/How to Test
<br> 

1. Log into Drupal
2. Go to the Digital Team page, or another department page on Boston.gov.
3. Make a change to the page (even just copying and pasting the title) and click save.
4. Confirm that you're also timing out.

_Reminder: Add any relevant Labels, Projects, and/or Milestone(s). If fields don't apply, please right N/A._

**Comment by duffy-james** [comment]
One note: tested this issue with someone who has content author's access and with my account, which has site admin access. The content author still saw the same timing out issue.

**Comment by jimafisk** [comment]
Thanks @duffy-james, I'm seeing the same issue as a super admin. If I open a duplicate tab while the original one is spinning, I do get the confirmation message and my changes are saved. 

![open_new_tab](https://user-images.githubusercontent.com/5913244/43152939-eca6c8f6-8f3d-11e8-8aeb-a9b06a6c12bb.png)

It's not desirable, but can you use this as a short term workaround while we work on a more permanent fix? Thanks!

**Comment by duffy-james** [comment]
@jimafisk in testing the Digital Team page, I removed the News & Announcements component and was able to save the page and view the draft. I then went and added an Upcoming Events component and was able to save the page as well and view the draft.
<br><br> I tested Upcoming Events because it relies on related content, just like the News & Announcements component. So, it looks like the issues is with the News & Announcements component.
